test-framework/test-suites/integration/tests/pytest.ini

```diff
@@ -1,4 +1,6 @@
 [pytest]
 usefixtures = 
+    add_host
     revert_database
     revert_filesystem
+    set_host_interface 
```

Comment:
This also probably doesn't belong here. Also, it doesn't appear be be used in any tests.